charts/kof-istio/templates/child-profile.yaml

```
@@ -0,0 +1,63 @@
+{{- if .Values.rootCA.enabled }}
+apiVersion: config.projectsveltos.io/v1beta1
+kind: ClusterProfile
+metadata:
+  name: istio-child
+spec:
+  clusterSelector:
+    matchLabels:
+      k0rdent.mirantis.com/istio-role: child
+  templateResourceRefs:
+    - identifier: Secret
+      resource:
+        apiVersion: v1
+        kind: Secret
+        name: {{ $.Release.Name }}-{{ `{{ .Cluster.metadata.name }}` }}-ca
+        namespace: {{ $.Release.Namespace }}
+  policyRefs:
+    - kind: ConfigMap
+      name: {{ $.Release.Name }}-secret-template
+      namespace: {{ $.Release.Namespace }}
+  helmCharts:
+    - repositoryName:   k0rdent-catalog
+      repositoryURL:    oci://ghcr.io/k0rdent/catalog/charts
+      chartName:        cert-manager
+      chartVersion:     1.16.2
+      releaseName:      cert-manager
+      releaseNamespace: kof
+      helmChartAction:  Install
+      values: |
+        cert-manager:
+          crds:
+            enabled: true
+
+    - repositoryName:   kof
+      repositoryURL:    {{ .Values.kcm.kof.repo.url }}
+      {{- if .Values.kcm.kof.repo.insecure }}
+      registryCredentialsConfig:
+        plainHTTP: true
+      {{- end }}
+      chartName:        kof-istio
+      chartVersion:     {{ .Chart.Version }}
+      releaseName:      kof-istio
+      releaseNamespace: istio-system
+      helmChartAction:  Install
+      values: |
+        rootCA:
+          enabled: false
+        intermediateCAs:
+          {{ `{{ .Cluster.metadata.name }}` }}:
+            certificate: false
+            issuer: true
+        global:
+          multiCluster:
+            clusterName: {{ `{{ .Cluster.metadata.name }}` }}
+          network: {{ `{{ .Cluster.metadata.name }}` }}-network
+        cert-manager-istio-csr:
+          app:
+            certmanager:
+              issuer:
+                name: kof-istio-{{ `{{ .Cluster.metadata.name }}` }}-ca
```

We expect release name to be `kof-istio`, but we should not hardcode it, especially as istio may move from kof to kcm in future:
```suggestion
                name: {{ $.Release.Name }}-{{ `{{ .Cluster.metadata.name }}` }}-ca
```